styles/autocomplete.less

```diff
@@ -16,10 +16,10 @@ atom-overlay.autocomplete-plus {
 }
 
 autocomplete-suggestion-list.select-list.popover-list {
-  width: auto;
+  width: auto !important; // TODO: Can be removed once the inline style is gone
```

Comment:
Had to override the width with `!important`. This could be removed once the inline style is gone.